test/Java8andUp/src_90/org/openj9/test/java/lang/Test_System.java

```diff
@@ -146,7 +146,8 @@ public void test_currentTimeMillis() {
 	public void test_getProperties() {
 		Properties p = System.getProperties();
  		AssertJUnit.assertTrue("Incorrect properties returned", p.getProperty(
- 				"java.version").indexOf("9", 0) == 0);
+ 				"java.version").indexOf("9", 0) == 0 || p.getProperty(
```

Comment:
What is the purpose of this test?  The code here will allow Java 10 to return 9 and Java 9 to return 10 and still have the test pass which clearly isn't the intention.